tests/mock/MockNetworkListener.h

```diff
@@ -59,7 +59,7 @@ class MockNetworkListener : public QTcpServer
     DEFLECT_API virtual ~MockNetworkListener();
 
 protected:
-    void incomingConnection(int handle) override;
+    void incomingConnection(int handle);
```

Comment:
yeah! just re-add final / override here :-)
I remember now that we had troubles in DisplayCluster with  "#define final" because some external function somewhere was called final(), which the preprocessor translated to (). But if it works here, all good!